collector/hikaricp/dashboards/overview/outputs.tf

```diff
@@ -0,0 +1,4 @@
+output "dashboard_url" {
```

Comment:
I like the files separate.
(1) can't be bothered by boilerplate, because laziness is a virtue and
(2) some tools that consume these modules only use the resource block